confdgnmi/testtool/gNMI_Interface/04__Subscribe.robot

```diff
@@ -5,90 +5,105 @@ Test Tags        subscribe
+
+Subscribe STREAM sends final message with "sync_response"
+    [Documentation]    When a STREAM subscription is created, the device must
+    ...    send an initial set of responses terminated by an empty
+    ...    response with "sync_response".
+    Given Subscription paths    ${GET-PATH}
+    And Subscription POLL with encoding JSON_IETF
+    Then Device sends terminated response series
 
 QOS marked subscriptions
+    [Tags]    unimplemented
+    # idk., what to test here?
 
 STREAM with TARGET_DEFINED mode
+    [Tags]    unimplemented
+    # same - nothing to test, really
 
 STREAM with ON_CHANGE mode
+    [Tags]    unimplemented
 
 STREAM with SAMPLE mode
-
-
-
-# TODO:
-
-Basic functionality of the Subscribe ONCE
-    [Documentation]    TODO
-    ...  Failure: Device does not respond, responds with an error, responds with
-    ...  an empty notification set or with a notification without updates,
-    ...  responds with incorrect encoding.
-    ...
-    ...  Tags: Rel-1, active, intf; Short: gNMI Subscribe ONCE Parent: GNMI-HLT-010
-    ...
-    ...  Test is passed path to the data model, which contains limited number of elements.
-    ...   ONCE Subscription operation is invoked.
-    ...  SubscriptionRequest does not have set updates_only field.
-    ...  Test verifies for each subscription
-    ...    data is received as a stream of SubscribeResponses
-    ...  Last SubscribeResponse is with sync_response set to true.
-    # TODO
-    # Can SubscribeResponse with sync_response contain also something else, e.g. updates?
+    [Tags]    unimplemented
 
 
 Subscribe ONCE with updates_only in the SubscriptionList
     # Failure: Device does not respond, responds with an error, responds with an empty notification set or with a notification without updates, responds with incorrect encoding.
     # Test is passed path to the data model, which contains limited number of elements. ONCE Subscription operation is invoked with updates_only field set in SubscriptionRequest. Test verifies only one SubscribeResponse with only sync_response set to true.
+    [Tags]    unimplemented
 
 Basic functionality of the Subscribe POLL RPC .
     # Parameters: common connection parameters, path, poll count, poll interval.
     # Failure: Device does not respond, responds with an error, responds with an empty notification set or with a notification without updates, responds with incorrect encoding.
     # Test is passed path to the data model, which contains limited number of elements. Subscription operation is invoked. First SubscriptionRequest is ONCE. After that POLL subscription requests are invoked with poll interval delay. Test verifies for each subscription data is received. Optionally, it can verify the data is the same as the one received in response for ONCE subscription. Test issues WARNING, if SubscriptionResponse stream is closed prematurely. Test issues WARNING if Updates in SubscriptionResponse are aggregated.
```

Comment:
`First SubscriptionRequest is ONCE.` - this is wrong, it should be `First SubscriptionRequest is with POLL mode in SubscriptionList`.